webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2022-04-19 14:38+0200\n"
+"POT-Creation-Date: 2022-04-19 15:26+0200\n"
```

Comment:
Maybe also unify the texts? Why do we have twice the exact same text for the "Zusammenveranlagung"?

Reply:
we could, yes